src/imitation/policies/serialize.py

```diff
@@ -230,3 +230,14 @@ def _on_step(self) -> bool:
         output_dir = self.policy_dir / f"{self.num_timesteps:012d}"
         save_stable_model(output_dir, self.model)
         return True
+
+
+def _load_rollouts_from_huggingface(
+    algo_name: str,
+    env_name: str,
+    organization: str = "HumanCompatibleAI",
+) -> str:
+    model_name = hfsb3.ModelName(algo_name, hfsb3.EnvironmentName(env_name))
+    repo_id = hfsb3.ModelRepoId(organization, model_name)
+    filename = hfsb3.load_from_hub(repo_id, "rollouts.pkl")
```

Comment:
I thought we were using `npz` format for rollouts now not pickle? As of https://github.com/HumanCompatibleAI/imitation/pull/448

Reply:
When I trained the RL agents, the rollouts were saved using the `npz` format but with the `pkl` extension, which got fixed in #591. Thanks for pointing this out. I'll rename the extensions on the hub and update the file.